blueprints/networking/glb-hybrid-neg-internal/landing-hub.tf

```diff
@@ -0,0 +1,247 @@
+      ]
+      rules = [{ protocol = "tcp", ports = [22] }]
+    }
+  }
+}
+
+module "nats_landing" {
+  for_each       = var.region_configs
+  source         = "../../../modules/net-cloudnat"
+  project_id     = module.project_landing.project_id
+  region         = each.value.region_name
+  name           = "nat-${each.value.region_name}"
+  router_network = module.vpc_landing_untrusted.self_link
+}
+
+module "nva_instance_templates" {
+  for_each               = var.region_configs
+  source                 = "../../../modules/compute-vm"
+  project_id             = module.project_landing.project_id
+  can_ip_forward         = true
+  create_template        = true
+  name                   = "nva-${each.value.region_name}"
+  service_account_create = true
+  zone                   = each.value.zone
+
+  metadata = {
+    startup-script = templatefile(
+      "${path.module}/data/nva-startup-script.tftpl",
+      {
+        gateway-trusted = cidrhost(module.vpc_landing_trusted.subnet_ips["${each.value.region_name}/trusted-${each.value.region_name}"], 1)
+        spoke-r1-subnet = module.vpc_spoke_01.subnet_ips["${var.region_configs.r1.region_name}/spoke-01-${var.region_configs.r1.region_name}"]
+        spoke-r2-subnet = module.vpc_spoke_01.subnet_ips["${var.region_configs.r2.region_name}/spoke-01-${var.region_configs.r2.region_name}"]
+      }
+    )
+  }
+
+  network_interfaces = [
+    {
+      network    = module.vpc_landing_untrusted.self_link
+      subnetwork = module.vpc_landing_untrusted.subnet_self_links["${each.value.region_name}/untrusted-${each.value.region_name}"]
+    },
+    {
+      network    = module.vpc_landing_trusted.self_link
+      subnetwork = module.vpc_landing_trusted.subnet_self_links["${each.value.region_name}/trusted-${each.value.region_name}"]
+    }
+  ]
+
+  tags = [
+    "http-server",
+    "https-server",
+    "ssh"
+  ]
+}
+
+module "nva_migs" {
+  for_each          = var.region_configs
+  source            = "../../../modules/compute-mig"
+  project_id        = module.project_landing.project_id
+  location          = each.value.zone
+  name              = "nva-${each.value.region_name}"
+  target_size       = 1
+  instance_template = module.nva_instance_templates[each.key].template.self_link
+}
+
+module "nva_untrusted_ilbs" {
+  for_each      = var.region_configs
+  source        = "../../../modules/net-ilb"
+  project_id    = module.project_landing.project_id
+  region        = each.value.region_name
+  name          = "nva-ilb-${each.value.region_name}"
+  service_label = "nva-ilb-${each.value.region_name}"
+  vpc_config = {
+    network    = module.vpc_landing_untrusted.self_link
+    subnetwork = module.vpc_landing_untrusted.subnet_self_links["${each.value.region_name}/untrusted-${each.value.region_name}"]
+  }
+  backends = [{
+    group = module.nva_migs[each.key].group_manager.instance_group
+  }]
+  health_check_config = {
+    tcp = {
+      port = 22
+    }
+  }
+}
+
+module "hybrid-glb" {
```

Comment:
Can you move the GLB into a `glb.tf` file?

Reply:
done